NERODesign/content/App.qml

```diff
@@ -5,14 +5,19 @@ import QtQuick 6.5
 import NERO
 
 Window {
-    width: mainScreen.width
-    height: mainScreen.height
+    width: 800
+    height: 480
+    color: "black"
 
     visible: true
     title: "NERO"
 
-    home {
-        id: mainScreen
+    Spedometer {
```

Comment:
so why is there an App.qml in NERODesign and NERODevelopment. Like what is the purpose of NERODesign vs NERODevelopment is what I'm asking

Reply:
theyre two projects, ones for qt design studio and the other one is for qt creator, you create the design in design studio then export it to creator